src/index.ts

```diff
@@ -51,6 +51,7 @@ export * from "./type.js";
 export * from "./uri.signing.js";
 export * from "./user.js";
 export * from "./vault.js";
+export * from "./origin.js";
```

Comment:
these lines are alphabetized - if you look above on line 37 you will see a line identical to the one you are adding.